### PR TITLE
New template, first version.

### PR DESCRIPTION
**Pull request by emmanuellujan**

### CODE REVIEW

**Comment by emmanuellujan** [comment]
Issue with `post_author = pagevar(surl, :post_author)` in `utils.jl`. When running `serve()` the variable `post_author` is sometimes set to `nothing`.  See [this issue](https://github.com/tlienart/Franklin.jl/issues/937).

**Comment by emmanuellujan** [comment]
@vchuravy with the exception of the issue I named above, I think this first version of the template is ready.

**Comment by sloede** [comment]
Out of curiosity: Is there a preview possible or would I have to clone and serve locally using Franklin.jl?

**Comment by emmanuellujan** [comment]
Hi!, I just added a screenshot, I don't have a demo online.

File: Project.toml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [deps]
 Franklin = "713c75ef-9fc9-4b05-94a9-213340da978e"
 NodeJS = "2bd173c7-0d6d-553b-b6af-13a54713934c"
+Xranklin = "558449b0-171e-4e1f-900f-d076a5ddf486"
```
Comment (giordano) [inline Project.toml:4]: Ooops

Comment (giordano) [reply Project.toml:4]: Or, if it refers to https://github.com/tlienart/Xranklin.jl I you need to check in the manifest (which for the website would be a good idea)

Comment (emmanuellujan) [reply Project.toml:4]: Follow-up here: https://github.com/JuliaParallel/juliaparallel.github.io/pull/4

**Pull request merged**